### PR TITLE
Docker roles

### PR DESCRIPTION
**Pull request by hi-a**

### CODE REVIEW

File: site/roles/trinity/docker-registry/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+- name: Render docker-registry systemd unit file
+  template:
+    src: 'docker-registry.service'
+    dest: '/usr/lib/systemd/system/docker-registry.service'
+  notify: restart docker-registry
+
+- name: Enable docker-registry service
+  file:
+# service:
```
Comment (kostty) [inline site/roles/trinity/docker-registry/tasks/main.yml]: doesn't `service`  work for enabling?

File: site/roles/trinity/ssl-cert/tasks/main.yml [hunk 2]
```diff
@@ -1,16 +1,21 @@
...
+      name: '{{ ssl_cert_group }}'
+      gid: '{{ ssl_cert_group_id }}'
+      system: yes
 
-- name: Ensure '{{ ssl_cert_group }}' group exists
-  group:
-    name: '{{ ssl_cert_group }}'
-    gid: '{{ ssl_cert_group_id }}'
-    system: yes
+  when: ansible_connection not in 'lchroot'
```
Comment (kostty) [inline site/roles/trinity/ssl-cert/tasks/main.yml]: maybe replace the double condition with something like `on_controller` set in the `init` role

File: site/roles/trinity/ssl-cert/tasks/main.yml [hunk 3]
```diff
@@ -1,16 +1,21 @@
...
+        and not compute|default(False)
```
Comment (kostty) [inline site/roles/trinity/ssl-cert/tasks/main.yml]: another _cosmetic_ comment: set `compute` equal to `False` in `defaults/main.yml` for clarity

**Comment by kostty** [comment]
looks good

**Comment by kostty** [comment]
still looks good

**Comment by hi-a** [comment]
jenkins test this please

**Comment by dchirikov** [comment]
Jenkins test this please

**Comment by dchirikov** [comment]
As Zabbix guys fixed their stuff:
https://support.zabbix.com/browse/ZBX-13761
I'll ask again
Jenkins test this please

**Pull request merged**